jobs/aws/eks-distro/main-1-18-postsubmits.yaml

```diff
@@ -35,11 +35,14 @@ postsubmits:
       containers:
       - name: build-container
         image: 316434458148.dkr.ecr.us-west-2.amazonaws.com/eks-distro/builder:afa151c76fdefe7248c1a75bb1a764c7e28fb334
+        env:
+        - name: TEST_ROLE_NAME
+          value: "TestBuildRole"
         command:
         - bash
         - -c
         - >
-          make postsubmit-conformance DEVELOPMENT=false RELEASE_BRANCH=1-18 IMAGE_TAG='$(GIT_TAG)-$(PULL_BASE_SHA)' ARTIFACT_BUCKET=artifactsstack-3794122512-artifactsbucket2aac5544-1f3dgu9wrpiz2
```

Comment:
where are we getting the artifact bucket now?

Reply:
Maybe you don't need the bucket, you just need the web site

Reply:
Artifacts bucket will be added as an env in the eks-distro main makefile.

Reply:
Okay, I have something going there. We'll see how it is received.

Reply:
Why don't we actually have it as an env here so we don't have it as a default for customers in the Makefile?

Reply:
Can we also have the main presubmit invoke something like a presubmit target rather than what it is right now? That way, we can have it invoked on the `cmd/` directory as well.